cmd/utils.go

```diff
@@ -954,3 +961,57 @@ func checksum256TestFile(newFileName string, oldFileName string) (bool, error) {
 
 	return checkValue, nil
 }
+
+func getLatestVersion() string {
+	resp, err := http.Get(LatestVersionURL)
+	if err != nil {
+		Warning.log("Unable to check the most recent version of Appsody in GitHub.... continuing.")
+	}
+	url := resp.Request.URL.String()
+	r, _ := regexp.Compile(`\d.\d.\d`)
+
+	return r.FindString(url)
+}
+
+func doVersionCheck(data []byte, old string, new string, file string) {
+	var latest = getLatestVersion()
+	if VERSION != "vlatest" && VERSION != latest {
+		Info.log("*\n*\n*\n\nA new CLI update is available.\nPlease go to " + LatestVersionURL + " and update from " + VERSION + " --> " + latest + ".\n\n*\n*\n*")
+	}
+	output := bytes.Replace(data, []byte(old), []byte(new), -1)
+	err := ioutil.WriteFile(file, output, 0666)
+	if err != nil {
+		Warning.log("Error writing to config file")
+	}
+}
+
+func getLastCheckTime() string {
+	return cliConfig.GetString("lastversioncheck")
+}
+
+func checkTime() {
+	var lastCheckTime string
+	var currentTime string
+
+	var configFile = getDefaultConfigFile()
```

Comment:
After talking to @skoh7645 - it makes sense to leave this value in the default config file, since the users are not going to meddle with it. Approving.